superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -123,6 +123,7 @@ const DEFAULT_WORKING_TIMEOUT = 3600;
 const DEFAULT_CRON_VALUE = '0 * * * *'; // every hour
 const DEFAULT_ALERT = {
   active: true,
+  creation_method: 'alerts_reports',
```

Comment:
@duynguyenhoang because this fixes a regression is it possible to add a unit test?